### PR TITLE
fixs typo: dont -> don't and Dont -> Don't

### PR DESCRIPTION
**Pull request by JoeWrightss**
fixs typo error: dont -> don't and Dont -> Don't
/assign @gnufied

### CODE REVIEW

**Comment by fejta** [comment]
Please combine all your test-infra typo PRs into a single PR.

**Comment by JoeWrightss** [comment]
@fejta Thank you for reminding me，I only have this typo PR yet.

File: vendor/github.com/docker/docker/api/types/stats.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ type NetworkStats struct {
 	RxBytes uint64 `json:"rx_bytes"`
 	// Packets received. Windows and Linux.
 	RxPackets uint64 `json:"rx_packets"`
-	// Received errors. Not used on Windows. Note that we dont `omitempty` this
+	// Received errors. Not used on Windows. Note that we don't `omitempty` this
```
Comment (justinsb) [inline vendor/github.com/docker/docker/api/types/stats.go]: This file is vendored, so you shouldn't change it here.

Looks like it was recently fixed upstream, so we should pick up this fix when we update the dependency: https://github.com/moby/moby/blob/master/api/types/stats.go

**Comment by justinsb** [comment]
We shouldn't be changing `vendor/` here, but otherwise LGTM

**Comment by fejta** [comment]
A quick grep (`ls | grep -v vendor | grep -v bazel | xargs grep -R Dont`) suggests this does not fix all instances of `Dont` in this repository.

As Justin mentions please do not change code in `vendor` except via the `hack/update-deps.sh` script.

Instead of fixing this one PR at a time, one repo at a time, can you create a `hack/verify-spelling.sh` or something along those lines? In fact, we already have this in k/k, perhaps it would be more valuable to move this code into this repo: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-spelling.sh

**Pull request closed without merging**